apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -47,6 +51,30 @@ ${context}
 `;
 }
 
+async function checkRender(
+  status: 'success' | 'error',
+  errors: string[],
+  courseId: string,
+  userId: string,
+  questionId: string,
+) {
+  if (status === 'success' && errors.length === 0) {
+    const question = await selectQuestionById(questionId);
+    const course = await selectCourseById(courseId);
+    const user = await selectUserById(userId);
+    const authnUser = await selectUserById(userId); //user id == authn user id in this case
```

Comment:
underlying assumption is that the authnUserId is fed into question generation as userId